.github/workflows/unity-nuget-test.yml

```
@@ -26,7 +26,8 @@ jobs:
           aws-region: us-east-1
       - name: Start Instance
         run: |
-          aws ec2 start-instances --instance-ids ${{ vars.UNITY_INSTANCE_ID }}
+          $instanceState = aws ec2 describe-instances --instance-ids ${{ vars.UNITY_INSTANCE_ID }} --query 'Reservations[].Instances[].State.Name' --output text
+          if ($instanceState -ne "running") { aws ec2 start-instances --instance-ids ${{ vars.UNITY_INSTANCE_ID }} }
```

```suggestion
          if ($instanceState -eq "stopped") { aws ec2 start-instances --instance-ids ${{ vars.UNITY_INSTANCE_ID }} }
```
if I am not mistaken, we can only start ec2 instance if it is in the "stopped" state
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-lifecycle.html